CMakeLists.txt

```diff
@@ -208,13 +219,35 @@ cmake_dependent_option(BUILD_SHARED_LIBS "Build shared libraries" ON
   "NOT JAEGERTRACING_PLUGIN" OFF)
 
 if(BUILD_SHARED_LIBS)
+  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
   add_library(jaegertracing SHARED ${SRC})
+#  target_compile_definitions(jaegertracing PRIVATE JAEGERTRACING_EXPORTS)
+if (WIN32)
+  target_link_libraries(jaegertracing PUBLIC Iphlpapi Ws2_32)
+endif()
   add_lib_deps(jaegertracing)
   set_target_properties(jaegertracing PROPERTIES
     VERSION ${PROJECT_VERSION}
     SOVERSION ${PROJECT_VERSION_MAJOR})
   set(JAEGERTRACING_LIB jaegertracing)
   list(APPEND JAEGERTRACING_LIBS jaegertracing)
+
+  if (MSVC)
+    # Copy the shared opentracing shared library next to jaeger tracing shared library
```

Comment:
I understand the reason for this, but IDK if this is the ideal solution. Please ask @ruslo in the Hunter gitter chat.

Reply:
I don't think it's related to Hunter, what is the exact reason of copying libraries?

Reply:
Well this project is on Hunter at the moment. I think there is some issue with Windows, but awaiting more detail from @mdouaihy.

Reply:
The problem I faced here is that the OT dynamic libraries were not accessible when running the Jaeger tests. On Linux, this works because of (or thanks to) the rpath.

Reply:
> dynamic libraries were not accessible when running the Jaeger tests

You can modify PATH environment variable instead of copying libraries, something like that:
* https://github.com/cgold-examples/fruits/blob/081f76a970d3ebbc8d316ad8ef52994b8b4cad56/test/fruits/skin_off/CMakeLists.txt#L8-L27

Reply:
ok, I will do that.

Reply:
Changed. The path env variable is not set for the tests.